src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -9,40 +9,52 @@ docs_area: manage
 {% include_cached feature-phases/limited-access.md %}
 {{site.data.alerts.end}}
 
-[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning of {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
+[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning, deprovisioning, and identity management for {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
 
 Rather than using [invitations](managing-access.html#invite-team-members-to-an-organization) or self-service [autoprovisioning](cloud-org-sso.html#autoprovisioning), SCIM autoprovisioning tasks are performed centrally by a team of IAM admins in your IdP, who manage the assignment of your organization's users to your organization's app integrations. This page describes SCIM provisioning and shows how to configure and use it to provision {{ site.data.products.db }} users.
 
 ## Overview of SCIM provisioning
 
 This section describes how SCIM provisioning works if you use Okta. Depending on how your IdP has implemented SCIM, some details may differ.
 
-1. In your IdP, an IAM admin creates a SCIM app integration and configures it to authenticate to {{ site.data.products.db }} using a {{ site.data.products.db }} service account with the [**Org Administrator (legacy)** role](authorization.html#org-administrator-legacy).
-1. In your IdP, an IAM admin assigns users and groups to the app integration.
-1. The app integration provisions or deactivates users in {{ site.data.products.db }} based on the app integration's assignments.
+If your IdP is Okta, then it may be helpful to read Okta's [article about SCIM](https://developer.okta.com/docs/concepts/scim/), as well as [Configure provisioning for an app integration
+](https://help.okta.com/en-us/Content/Topics/Provisioning/lcm/lcm-provision-application.htm) in the Okta documentation. Otherwise, refer to your IdP's documentation about configuring SCIM.
```

Comment:
```suggestion
If your IdP is Okta, then it may be helpful to read Okta's [article about SCIM(https://developer.okta.com/docs/concepts/scim/), as well as [Configure provisioning for an app integration
](https://help.okta.com/en-us/Content/Topics/Provisioning/lcm/lcm-provision-application.htm) in the Okta documentation. Otherwise, refer to your IdP's documentation about configuring SCIM.
```